tests/test-cases/include-remote-with-inner-local/integration.include-remote-with-inner-local.test.ts

```diff
@@ -8,7 +8,7 @@ test("include-remote-with-inner-local", async () => {
     const writeStreams = new MockWriteStreams();
     const spyGitRemote = {
         cmd: "git remote -v",
-        returnValue: {stdout: "origin\tgit@gitlab.com:gcl/include-remote-with-inner-local.git (fetch)\norigin\tgit@gitlab.com:gcl/include-remote-with-inner-local.git (push)\n"},
+        returnValue: {stdout: "origin\tgit@gitlab.com:aepfli/include-remote-with-inner-local.git (fetch)\norigin\tgit@gitlab.com:aepfli/include-remote-with-inner-local.git (push)\n"},
```

Comment:
@firecow should this be gcl? or should this be firecow? i do not have access to gitlab.com/gcl :) if we leave it with `gcl` this will most likely affect everyone. so i suggest to either set it to `firecow` or make `gcl` public accessible

Reply:
I'm just mocking the remote host with the gcl thingie...
It's still fetching from gitlab.com/firecow/gitlab-ci-local-includes
There is no need to change this test or its matching .gitlab-ci.yml

Reply:
well if i read the code correctly this will end up in a remote like `gitlab.com:gcl/firecow/gitlab-ci-local-includes` which is an url like `gitlab.com/gcl/firecow.gitlab-ci-local-includes` (gitlab also allows group names as ports) 

seen here
https://github.com/firecow/gitlab-ci-local/blob/master/src/parser-includes.ts#L120

Reply:
```suggestion
        returnValue: {stdout: "origin\tgit@gitlab.com:firecow/include-remote-with-inner-local.git (fetch)\norigin\tgit@gitlab.com:firecow/include-remote-with-inner-local.git (push)\n"},
```

Reply:
but i realize this should than also work with other tests... maybe i derped